web/src/pages/api/public/dataset-run-items.ts

```diff
@@ -49,14 +51,20 @@ export default withMiddlewares({
 
       // Backwards compatibility: historically, dataset run items were linked to observations, not traces
       if (!traceId && observationId) {
-        const observation = observationId
-          ? await prisma.observation.findUnique({
-              where: {
-                id: observationId,
-                projectId: auth.scope.projectId,
-              },
-            })
-          : undefined;
+        const observation =
+          env.LANGFUSE_RETURN_FROM_CLICKHOUSE === "true"
```

Comment:
Nit, global: we could extract a small helper that returns if env variables are true or false -- less error prone generally

Reply:
Created https://linear.app/langfuse/issue/LFE-2948/consider-an-env-helper-function-to-avoid-error-prone-===-true-lookup as a tracking issue.